src/pkg/scan/vuln/severity.go

```diff
@@ -37,3 +37,21 @@ const (
 
 // Severity is a standard scale for measuring the severity of a vulnerability.
 type Severity string
+
+// Code returns the int code of the severity for comparing.
+func (s Severity) Code() int {
+	switch s {
+	case Negligible:
+		return 0
+	case Low:
+		return 1
+	case Medium:
+		return 2
+	case High:
+		return 3
+	case Critical:
+		return 4
+	default:
+		return -1
```

Comment:
See my comment regarding how we handle "Unknown"

Reply:
Agree with this comment, let's assign it the highest code.